vendor/windows_game_input/windows_game_input.odin

```diff
@@ -0,0 +1,1129 @@
+	mouseInfo: ^GameInputMouseInfo,
+	touchSensorInfo: ^GameInputTouchSensorInfo,
+	motionInfo: ^GameInputMotionInfo,
+	arcadeStickInfo: ^GameInputArcadeStickInfo,
+	flightStickInfo: ^GameInputFlightStickInfo,
+	gamepadInfo: ^GameInputGamepadInfo,
+	racingWheelInfo: ^GameInputRacingWheelInfo,
+	uiNavigationInfo: ^GameInputUiNavigationInfo,
+	forceFeedbackMotorInfo: ^GameInputForceFeedbackMotorInfo,
+	hapticFeedbackMotorInfo: ^GameInputHapticFeedbackMotorInfo,
+	displayName: ^GameInputString,
+	deviceStrings: ^GameInputString,
+	deviceDescriptorData: rawptr,
+}
+
+GameInputFlightStickInfo :: struct {
+	menuButtonLabel: GameInputLabel,
+	viewButtonLabel: GameInputLabel,
+	firePrimaryButtonLabel: GameInputLabel,
+	fireSecondaryButtonLabel: GameInputLabel,
+	hatSwitchKind: GameInputSwitchKind,
+}
+
+GameInputFlightStickState :: struct {
+	buttons: GameInputFlightStickButtons,
+	hatSwitch: GameInputSwitchPosition,
+	roll: f32,
+	pitch: f32,
+	yaw: f32,
+	throttle: f32,
+}
+
+GameInputForceFeedbackConditionParams :: struct {
+	magnitude: GameInputForceFeedbackMagnitude,
+	positiveCoefficient: f32,
+	negativeCoefficient: f32,
+	maxPositiveMagnitude: f32,
+	maxNegativeMagnitude: f32,
+	deadZone: f32,
+	bias: f32,
+}
+
+GameInputForceFeedbackConstantParams :: struct {
+	envelope: GameInputForceFeedbackEnvelope,
+	magnitude: GameInputForceFeedbackMagnitude,
+}
+
+GameInputForceFeedbackEnvelope :: struct {
+	attackDuration: u64,
+	sustainDuration: u64,
+	releaseDuration: u64,
+	attackGain: f32,
+	sustainGain: f32,
+	releaseGain: f32,
+	playCount: u32,
+	repeatDelay: u64,
+}
+
+GameInputForceFeedbackMagnitude :: struct {
+	linearX: f32,
+	linearY: f32,
+	linearZ: f32,
+	angularX: f32,
+	angularY: f32,
+	angularZ: f32,
+	normal: f32,
+}
+
+GameInputForceFeedbackMotorInfo :: struct {
+	supportedAxes: GameInputFeedbackAxes,
+	location: GameInputLocation,
+	locationId: u32,
+	maxSimultaneousEffects: u32,
+	isConstantEffectSupported: bool,
+	isRampEffectSupported: bool,
+	isSineWaveEffectSupported: bool,
+	isSquareWaveEffectSupported: bool,
+	isTriangleWaveEffectSupported: bool,
+	isSawtoothUpWaveEffectSupported: bool,
+	isSawtoothDownWaveEffectSupported: bool,
+	isSpringEffectSupported: bool,
+	isFrictionEffectSupported: bool,
+	isDamperEffectSupported: bool,
+	isInertiaEffectSupported: bool,
+}
+
+GameInputForceFeedbackParams :: struct {
+	kind: GameInputForceFeedbackEffectKind,
+	using _: struct #raw_union {  
+	constant: GameInputForceFeedbackConstantParams,
+	ramp: GameInputForceFeedbackRampParams,
+	sineWave: GameInputForceFeedbackPeriodicParams,
+	squareWave: GameInputForceFeedbackPeriodicParams,
+	triangleWave: GameInputForceFeedbackPeriodicParams,
+	sawtoothUpWave: GameInputForceFeedbackPeriodicParams,
+	sawtoothDownWave: GameInputForceFeedbackPeriodicParams,
+	spring: GameInputForceFeedbackConditionParams,
+	friction: GameInputForceFeedbackConditionParams,
+	damper: GameInputForceFeedbackConditionParams,
+	inertia: GameInputForceFeedbackConditionParams,
```

Comment:
Lack of indentation.

Reply:
Not your fault, but this is very weird in that they are all the same type but just different names. Seems kind of dumb.

Reply:
Again probably better to keep to be in sync with documentation